account_financial_risk/models/account_invoice.py

```diff
@@ -64,9 +64,8 @@ def _first_invoice_exception_msg(self):
         Method used to return the first invoice with exception message.
         """
         ret = False, False
-        if (
-            self.env.context.get("bypass_risk", False)
-            or self.company_id.allow_overrisk_invoice_validation
+        if self.env.context.get("bypass_risk", False) or all(
+            self.company_id.mapped("allow_overrisk_invoice_validation")
         ):
             return ret
         for invoice in self.filtered(lambda x: x.move_type == "out_invoice"):
```

Comment:
Try don't repeat loop
```suggestion
        if self.env.context.get("bypass_risk", False):
            return ret
        for invoice in self.filtered(lambda x: x.move_type == "out_invoice" and not x.company_id.allow_overrisk_invoice_validation):
```